### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by kargapolov**
Simple and concise.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, LCODE team
```
Comment (Korsidar) [inline LICENSE:3]: Add names?

**Comment by Korsidar** [comment]
I need more time to read about other licenses. I thought about GPL-3.

**Comment by kargapolov** [comment]
> I need more time to read about other licenses. I thought about GPL-3.

I picked this license based on several examples: https://github.com/fbpic/fbpic/blob/dev/LICENSE.txt https://github.com/Hi-PACE/hipace/blob/development/license.txt https://github.com/mikgroup/sigpy/blob/master/LICENSE

File: LICENSE [hunk 2]
```diff
@@ -27,3 +25,13 @@ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
...
+
+You are under no obligation whatsoever to provide any bug fixes, patches,
+or upgrades to the features, functionality or performance of the source
+code ("Enhancements") to anyone; however, if you choose to make your
+Enhancements available publicly without imposing a separate written
+license agreement for such Enhancements, then you hereby grant the following
+license: a non-exclusive, royalty-free perpetual license to install, use,
+modify, prepare derivative works, incorporate into other computer software,
+distribute, and sublicense such enhancements or derivative works thereof,
+in binary and source code form.
```
Comment (Korsidar) [inline LICENSE:37]: Ok

**Review by Korsidar — APPROVED** [review]

**Review by gornalexander — APPROVED** [review]
Ok

**Pull request merged**